HHS/Head-Start-TTADP
# Configure smtp host

## Description of change
Enables SMTP configuration. Fixes issues with directly accessing reports through links in the notifications.

NOTE - This is a prerequisite for the notifications being functional, but it needs to be deployed to prod once the ability to subscribe/unsubscribe is implemented.


## How to test
This is currently deployed to Sandbox. The server & the configuration can be tested there by adding a collaborator, approving a report, requesting changes, and submitting a report for approval.
https://tta-smarthub-sandbox.app.cloud.gov/

For example:
- create a report and choose yourself as the approver
- approve a report
- verify that you received a notification

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-756


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
